packages/nimble-components/src/select/tests/select-matrix.stories.ts

```diff
@@ -23,9 +23,9 @@ import {
 
 /* array of state name, invalidClass, errorText */
 const selectInvalidStates = [
-    ['', ''],
-    ['invalid', 'This is not valid.'],
-    ['invalid', '']
+    [false, ''],
+    [true, 'This is not valid.'],
+    [true, '']
 ] as const;
 type SelectInvalidState = typeof selectInvalidStates[number];
 
```

Comment:
We now have some shared errorStates that are used by the other components using the ErrorPattern that we should leverage: https://github.com/ni/nimble/blob/2c07a38eec88d0e88977f407bdef98ada6ca8ee7/packages/nimble-components/src/utilities/tests/states.ts#L29-L34